TransactionTypeApiConnector

tychokamphuis-accommodatiehuur opened the pull request:
Added functionality to get TRS -> Transaction types (daybooks) from Twinfield

rojtjo commented:
Thank you for your pull request!

I have reworked the [connector](https://github.com/php-twinfield/twinfield/blob/551238c44f1de5b7f5f259cfdfef65281faa8edd/src/ApiConnectors/TransactionTypeApiConnector.php) a little bit to include the correct options as specified in the docs. I also renamed the class from `TransactiontypeApiConnector` to `TransactionTypeApiConnector` and added a few tests.

tychokamphuis-accommodatiehuur commented:
> Thank you for your pull request!
> 
> I have reworked the [connector](https://github.com/php-twinfield/twinfield/blob/551238c44f1de5b7f5f259cfdfef65281faa8edd/src/ApiConnectors/TransactionTypeApiConnector.php) a little bit to include the correct options as specified in the docs. I also renamed the class from `TransactiontypeApiConnector` to `TransactionTypeApiConnector` and added a few tests.

Hi Rojtjo, thank you for your quick response. In the future I'll make sure to provide some tests.
